src/couch_replicator_scheduler.erl

```diff
@@ -266,57 +260,24 @@ remove_job_int(#job{} = Job) ->
     ets:delete(?MODULE, Job#job.id).
 
 
--spec job_count() -> non_neg_integer().
-job_count() ->
-    ets:info(?MODULE, size).
-
-
 -spec running_job_count() -> non_neg_integer().
 running_job_count() ->
-    MatchSpec = [{#job{pid = '$1', _='_'},
-        [{is_pid,'$1'}],
-        [true]}
-    ],
-    ets:select_count(?MODULE, MatchSpec).
-
+    ets:info(?MODULE, size) - pending_job_count().
 
 -spec running_jobs() -> [#job{}].
 running_jobs() ->
-    MatchSpec = [{#job{pid = '$1', _='_'},
-        [{is_pid,'$1'}],
-        ['$_']}
-    ],
-    ets:select(?MODULE, MatchSpec).
+    ets:tab2list(?MODULE) -- pending_jobs().
```

Comment:
Yeah, it is rather costly. I don't think you could use `ets:match_object` since we want to filter by type. The matchspec/select which I reverted is less costly, so I could un-revert that.